fedmsg_meta_fedora_infrastructure/centos_ci.py

```diff
@@ -41,26 +41,32 @@ def subtitle(self, msg, **config):
         namespace = msg['msg']['namespace']
         status = msg['msg']['status']
 
-        if 'ci.pipeline.package.ignore' in msg['topic']:
-            tmpl = self._(
-                'Commit "{commit}" of package {ns}/{pkg} is being '
-                'ignored by the CI pipeline on branch {branch}')
-
-        elif 'ci.pipeline.complete' in msg['topic']:
+        def _get_status(status):
             if status.lower() == 'success':
                 status = 'passed'
             elif status.lower() == 'aborted':
                 status = 'was aborted on'
             else:
                 status = 'failed'
+            return status
+
+        if 'ci.pipeline.package.ignore' in msg['topic']:
+            tmpl = self._(
+                'Commit "{commit}" of package {ns}/{pkg} is being '
+                'ignored by the CI pipeline on branch {branch}')
+
```

Comment:
Suggestion: dropping this empty line will make the ```elif``` more visually obvious, but on the flip side I see that this is the established convention in this file...

Reply:
yeah I've used the line spacing to give a little more space to the code as it is otherwise a rather long and hard(er imho) to read blob